x/leverage/keeper/collateral.go

```diff
@@ -20,10 +20,10 @@ func (k Keeper) burnCollateral(ctx sdk.Context, addr sdk.AccAddress, coin sdk.Co
 	return k.setUTokenSupply(ctx, k.GetUTokenSupply(ctx, coin.Denom).Sub(coin))
 }
 
-// removeCollateral removes some uTokens in fromAddr's collateral and sends them to toAddr. This
-// occurs when decollateralizing uTokens (in which case fromAddr and toAddr are the same) as well as
-// during liquidations, where toAddr is the liquidator.
-func (k Keeper) removeCollateral(ctx sdk.Context, fromAddr, toAddr sdk.AccAddress, coin sdk.Coin) error {
+// decollateralize removes fromAddrs' uTokens from circulation and redeems it for the underlying
+// token, sending it to the toAddr. It occurs when decollateralizing uTokens (in which case
+// fromAddr and toAddr are the same) as well as during liquidations, where toAddr is the liquidator.
```

Comment:
This is incorrect - the function does not redeem uTokens for underlying base tokens, but simply moves uTokens.